src/components/Accounts/Actions/CancellationModal.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Field, reduxForm } from 'redux-form';
 import { FormattedMessage } from 'react-intl';
+import { Field, reduxForm } from 'redux-form';
```

Comment:
let place multiple importing variables on the separate lines
and please, add a line break between 'redux-form' import and @folio-imports